api_schema/schema.go

```diff
@@ -56,8 +57,8 @@ type CreateLearnSessionRequest struct {
 	BaseAPISpecRef *APISpecReference `json:"base_api_spec_ref,omitempty"`
 
 	// Optional key-value pairs to tag this learn session.
-	// We reserve tags with "x-akita" prefix for internal use.
-	Tags map[string]string `json:"tags,omitempty"`
+	// We reserve tags with "x-akita" and "akita:" prefix for internal use.
```

Comment:
Did you decide to use `akita:` after all, or should we revert this comment?

Reply:
Fixed in d858e0c.